tests/test_sft_trainer.py

```diff
@@ -159,6 +201,7 @@ def test_parse_arguments_peft_method(job_config):
         parser, job_config_lora
     )
     assert isinstance(tune_config, peft_config.LoraConfig)
+    assert "target_modules" not in job_config_lora
```

Comment:
can also `assert not hassattr(tune_config, "target_modules")` so that it verifies the input from job_config_lora and this above assertion verifies that after parse_arguments run, the value is still None for target_modules

Reply:
When I add this check I get the following error:
```
>       assert not hasattr(tune_config, "target_modules")
E       AssertionError: assert not True
E        +  where True = hasattr(LoraConfig(r=8, lora_alpha=32, target_modules=None, lora_dropout=0.05), 'target_modules')
```
I think this is because the attribute target_modules exists but is None. Would this suffice?
`assert tune_config.target_modules is None`

Reply:
yes that makes sense and yes that check looks good! i think you can refactor it down to `assert not tune_config.target_modules`